src/hal/components/panelui.c

```diff
@@ -109,7 +109,7 @@ int nochange = 0x40;
 int rowshift;
 int ncols = 8;
 int nrows = 8;
-int rollover = 2; //maximuim number of simultaneous keys presses recongnised
+int rollover = 2; //maximuim number of simultaneous keys presses recognised
```

Comment:
I wonder if we should change that to the AE spelling like the majority of the documentation is written in?